packages/vite-plugin-svelte-inspector/src/runtime/Inspector.svelte

```diff
@@ -134,67 +136,67 @@
 		}
 	}
 
-	function is_key_active(key, event) {
+	function is_active(key, e) {
 		switch (key) {
 			case 'shift':
 			case 'control':
 			case 'alt':
 			case 'meta':
-				return event.getModifierState(key.charAt(0).toUpperCase() + key.slice(1));
+				return e.getModifierState(key.charAt(0).toUpperCase() + key.slice(1));
 			default:
-				return key === event.key.toLowerCase();
+				return key === e.code.replace(/^Key/, '').toLowerCase() || key === e.key.toLowerCase();
```

Comment:
this change helps detecting the physical key if the modifier changed what event.key is. (eg alt-x produces a double-tilde on mac)

took the liberty to slightly refactor the is_ helpers below to all use this logic to avoid inconsistencies.
Using toLowerCase allows the user to use mixed casing in their config, `Alt-X`, `alt-x` and `alt-X` all work the same